README.md

```diff
@@ -50,3 +50,5 @@ Include the following in a new line below `"scripts": {`:
 ```
 "preinstall": "npm install --save node@12.16.3",
 ```
+
+If you want the web UI, you will need to replace `server.js` in the start script with `bot-web.js`.
```

Comment:
The start script should actually not be modified. To start the web UI, you'll need to modify the `DRSS_START` env variable:

![image](https://user-images.githubusercontent.com/18688793/82736025-6f363c00-9cf4-11ea-889d-683e717cb8ae.png)

I chose not to make it `bot-web` by default because the user would want to get this working as soon as possible without having to deal with required web config errors, if it's set to `web` or `bot-web`

Reply:
Ok I didn't know that thanks, just updated